### PR TITLE
Added Rarimo's Proof of Humanity miniapp

### PR DESCRIPTION
**Pull request by ihordiachenko**

### CODE REVIEW

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<!-- TODO: replace with the prod link once the miniapp is live -->
+For detailed instructions, see [Protecting your Discord server from bots with Proof of Humanity and Collab.Land
+](https://staging.docs.rarimo.com/how-to-guides/proof-of-humanity-collabland-discord) guide.
+
+### Verifying Your Humanity
+
+1. Call the `/verify` command and follow the instructions.
+1. Complete the verification process at [Verify Your Humanity](https://robotornot.rarimo.com).
+1. Claim the role by verifying your assets in the `#collabland-join` channel.
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:37]: same process as usual {link}

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 11
+title: Proof of Humanity
+---
+
+### About this app
+
+Rarimo’s Proof-of-Humanity plug-in ensures that only verified humans are present either within a server or channel. It keeps the community server spam and scam-free, which saves mods’ time, improves the quality of conversation, and helps protect airdrops.
+
+The solution is uniquely robust yet does not require users to reveal any personal details. Users can select between five identity providers to verify their humanity and will receive a verified human credential that they can use across Web3 for perks and exclusive access.
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:10]: looks like the flow involves an NFT on polygon for PoH access. please describe a bit about it here to clarify for users. a direct link to your docs on the topic would be helpful also

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Call the `/verify` command and follow the instructions.
+1. Complete the verification process at [Verify Your Humanity](https://robotornot.rarimo.com).
+1. Claim the role by verifying your assets in the `#collabland-join` channel.
+
+## Commands
+
+| Command               | Description                                                                                                                                                                                   |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `/verify`  | Redirects you to Proof of Humanity DApp for verification.                                                                                                                          |
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:43]: > Redirects member

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Support
+
+- Discord: [Rarimo](https://discord.gg/cfrH3Fe7ke)
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:47]: make sure Discord invite [will not expire](https://support.discord.com/hc/en-us/articles/208866998-Invites-101#h_01H2DMMG355MV7GJDQ9F65SDVG), default invites expire 7 days from creation

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+1. Create the following token-gating rule in the Command Center:
+
+    | Field  | Value |
+    |---|---|
+    |Chain|`Polygon`|
+    |Token Type|`ERC721`|
+    |Addresses|`0xaD7De01cb7eaAFf3a419A0a0a3133a964cD90373`|
+
+1. Create a role-exclusive channel for verified humans.
+1. Ask users to verify their humanity using the `/verify` command, ideally in the `#collabland-join` channel.
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:27]: > Ask members

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+### About this app
+
+Rarimo’s Proof-of-Humanity miniapp ensures that only verified humans are present either within a server or channel. It keeps the community server spam and scam-free, which saves mods’ time, improves the quality of conversation, and helps protect airdrops.
+
+The solution is uniquely robust yet does not require users to reveal any personal details. Users can select between five identity providers to verify their humanity and will receive a verified human credential that they can use across Web3 for perks and exclusive access.
+
+### Getting Started
+
+To get started:
+
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:15]: Make explicit that these are actions for _server admins_ to take

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+1. Install the Proof of Humanity Miniapp from the [Collab.Land Marketplace](https://cc.collab.land/).
+1. Create a discord role for verified humans.
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:17]: Discord

**Review by iSpeakNerd — CHANGES_REQUESTED** [review]
- [x] make sure Discord invite [will not expire](https://support.discord.com/hc/en-us/articles/208866998-Invites-101#h_01H2DMMG355MV7GJDQ9F65SDVG), default invites expire 7 days from creation
- [x] make steps more explicit which persona is taking actions: **_Admin_** vs. **_Member_**
- [x] add bit more description at top about PoH process and link to Rarimo docs directly 
- [x] wrap Field names in \` in the commands table
- [x] change "user" to **_Member_** or **_Admin_**, depending on persona
- [x] add 1-2 sentences marketing blurb before `### about this app` for link social preview

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 16
+title: Proof of Humanity
+---
+
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:5]: add 1-2 sentences marketing blurb for link social preview

File: help-docs/marketplace/apps/proof-of-humanity.md (hunk 4 above)
Comment (ihordiachenko) [reply help-docs/marketplace/apps/proof-of-humanity.md:47]: it's a permalink

File: help-docs/marketplace/apps/proof-of-humanity.md (hunk 1 above)
Comment (ihordiachenko) [reply help-docs/marketplace/apps/proof-of-humanity.md:37]: Could you elaborate?

Comment (iSpeakNerd) [reply help-docs/marketplace/apps/proof-of-humanity.md:41]: that was a note for me to add something then i forgot to add it 🤣 my b

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+Learn more about [Proof of Humanity](https://docs.rarimo.com/use-cases/proof-of-humanity) by Rarimo.
+
+### Getting started
+
+To get started, the server Admin should follow these steps:
+
+1. Install the Proof of Humanity Miniapp from the [Collab.Land Marketplace](https://cc.collab.land/).
+1. Create a Discord role for verified humans.
+1. Create the following [token-gating rule](/help-docs/key-features/token-gate-communities) in the Command Center:
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/proof-of-humanity.md:24]: in a later app version, would be great if the miniapp did this TGR creation for the admins. is that possible @raymondfeng?

**Review by iSpeakNerd — APPROVED** [review]

File: help-docs/marketplace/apps/proof-of-humanity.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+title: Proof of Humanity
+---
+
+Keep your Discord server bot free using Rarimo’s Proof-of-Humanity plug-in
+
+### About this app
+
+Rarimo’s Proof-of-Humanity miniapp ensures that only verified humans are present either within a server or channel. It keeps the community server spam and scam-free, which saves mods’ time, improves the quality of conversation, and helps protect airdrops.
+
+Members can select between five identity providers to verify their humanity and will receive a verified human credential in their [RariMe extension for MetaMask](https://rarime.com/) that they can use across Web3 for perks and exclusive access. This credential will be used to generate a zero-knowledge proof that permits minting a Proof of Humanity SBT on Polygon. This SBT is then used for token gating.
```
[the remark this answers is not in the document]
Comment (ihordiachenko) [reply help-docs/marketplace/apps/proof-of-humanity.md]: We try not to emphasize the SBT part too much, it's just a way to integrate with existing platforms like Collab.Land. RariMe snap is the main star of the show

**Pull request merged**